frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -78,7 +78,7 @@
   "What is Caching?": "What is Caching?",
   "Caching is a policy that creates local copies of the data. It saves the copies locally to improve performance for frequently accessed data. Each cached copy has a TTL and is verified against the hub. Each non-read operation(upload, overwrite, delete) is performed on the hub": "Caching is a policy that creates local copies of the data. It saves the copies locally to improve performance for frequently accessed data. Each cached copy has a TTL and is verified against the hub. Each non-read operation(upload, overwrite, delete) is performed on the hub",
   "Hub namespace store ": "Hub namespace store ",
-  "A single namespace-store, defines the read and write target of the namespace bucket": "A single namespace-store, defines the read and write target of the namespace bucket",
+  "A single NamespaceStore which defines the read and write target of the namespaced bucket.": "A single NamespaceStore which defines the read and write target of the namespaced bucket.",
```

Comment:
A single Namespace store that defines the read and write target of the namespace bucket.